bin/publish-exocom

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
Just name this script `publish`, to match the [o-tools](https://github.com/originate/o-tools-node) spec